docs/dockerhub.md

```diff
@@ -51,6 +51,7 @@ Usage:
   kics scan [flags]
 
 Flags:
+  -m, --bom                           include bill of materials (BoM) in results output
       --cloud-provider strings        list of cloud providers to scan (aws, azure, gcp)
       --config string                 path to configuration file
```

Comment:
In the commands.md you included
`--disable-secrets               disable secrets scanning`
Shouldn't this be here as well?

Reply:
it's already there https://github.com/Checkmarx/kics/blob/aaf943b1d016b24149f0c39383a14871b1424b56/docs/dockerhub.md

Reply:
My bad